GitUI/ScriptsEngine/IScriptOptionsProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GitUI.ScriptsEngine;
+
+public interface IScriptOptionsProvider
+{
+    /// <summary>
+    /// Retrieves the information for placeholders in script arguments.
+    /// </summary>
+    /// <param name="option">The option identifier which is to be replaced.</param>
+    /// <returns>The value to be used for the script argument.</returns>
+    string GetValue(string option);
```

Comment:
Rather let the ScriptOptionsParser handle unsupported options:
```suggestion
    bool TryGetValue(string option, out string value);
```